install/kubernetes/helm/istio/charts/pilot/values.yaml

```diff
@@ -52,3 +52,8 @@ podAntiAffinityTermLabelSelector: []
 # to a pilot. It balances out load across pilot instances at the cost of
 # increasing system churn.
 keepaliveMaxServerConnectionAge: 30m
+
+# list of platform service registries to read from
+# options are Counsul, Kubernetes, MCP, Mock
+registries:
```

Comment:
Also: I don't think this is right here. Consul should be deprecated, Mock is not for normal install but tests, MCP registry should be automatically enabled and depend on mesh config. 

We might need an option to turn off loading endpoints from kubernetes - but I don't think this is the right place to config that. Maybe we don't even need such config - if MCP endpoint config sources are defined in mesh config we can assume primary k8s endpoint source is not used ? Keep in mind that k8s is also auto-added for remote apiservers, and we may have a mix of apiserver and MCP sources.

Reply:
This follows the pattern we have in pilot today to configure this property. I agree with the changes mentioned above, however It would make more sense if I created an issue for that and do the work as a follow up to this PR. Reason for that is the scope of this PR is quite large and I’m trying to avoid any further changes that makes testing of this feature harder.